R/calcBP.R

```diff
@@ -143,9 +143,9 @@ calcBP <- function() {
   weights[,,"US$2005/GJ", pmatch = T] <- 1
 
   return(list(
-    x = x, 
+    x = x,
     weight = weights,
-    mixed_aggregation = T, 
+    mixed_aggregation = T,
```

Comment:
TRUE, instead of T ?

Reply:
Correct, no idea why this wasn't raised by the linter (probably because, technically I did not change the file, just some whitespaces and line endings were changed when saving it). I will take care of getting these functions linter-conform the next time I actually touch them.